packages/front/src/lib/components/SubventionsVersementsDashboard/SubventionsVersementsDashboard.svelte

```diff
@@ -55,14 +49,16 @@
                     options={$exercicesOptions} />
             {/if}
         </div>
-        <Button
-            type="tertiary"
-            outline={false}
-            ariaControls="fr-modal"
-            on:click={displayModal}
-            trackerName="association-etablissement.dashboard.display-provider-modal">
-            Voir la liste des fournisseurs de données
-        </Button>
+        <div class="align-bottom">
+            <a
+                on:click={controller.clickProviderLink}
```

Comment:
T'as check que yavais pas de pb de `this` comme ça ? (je m'emmêle souvent les pinceaux sur ça donc je suis parano ><)

Reply:
Effectivement il pourrait y avoir un problème de `this`. Mais vu qu'on utilise aucunement `this` dans la méthode ça fonctionne très bien